tests/service/data-util.service.spec.ts

```diff
@@ -46,5 +46,25 @@ describe('Data Utils service test', () => {
         it('should return the bytesize of the text', inject([JhiDataUtils], (service: JhiDataUtils) => {
             expect(service.byteSize('Hello Jhipster')).toBe(`10.5 bytes`);
         }));
+
+        it('should download the csv file', inject([JhiDataUtils], (service: JhiDataUtils) => {
+            // create spy object with a click() method
+            const spyObj = spyOn('a', 'click');
```

Comment:
where do you get the `spyOn` method?  I dont see any import for it

Reply:
also did you run the test locally first? using `npm test`

Reply:
ok it seems like a global method. But `spyOn('a', 'click');` doesnt seem rite,  spyOn works on obejcts so should be something like `spyOn(element, 'onClick');

Reply:
spyOn is a Jest function. Sorry I think that ng-jhipster uses Jest for unit tests !